hbase-client/src/main/java/org/apache/hadoop/hbase/filter/ParseFilter.java

```diff
@@ -109,6 +109,7 @@ public Filter parseFilterString(String filterString) throws CharacterCodingExcep
    * @param filterStringAsByteArray filter string given by the user
    * @return filter object we constructed
    */
+  @SuppressWarnings("JdkObsolete")
```

Comment:
Removing the usage of Stack needs extra works as the API is different, but I think we should change it to something like ArrayDeque. So let's not add this SuppressWarnings annotation, just leave it as is for now and open another issue for fixing this, something like 'Remove the usage of java.util.Stack in our code base'

Reply:
Ok, I will remove it.